### PR TITLE
lesson8 pull req

### PR DESCRIPTION
**Pull request by saumya665**

### CODE REVIEW

**Review by MadhuBeesetty — APPROVED** [review]

File: lesson8/public/index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+/* camera*/
+const video = document.getElementById('vid');
+navigator.mediaDevices.getUserMedia({
+  video: true,
+  audio: false
+}).then((mediaStream) => {
+  video.srcObject = mediaStream;
+  video.play();
+}).catch((err)=>{
+  console.log(err);
```
Comment (bramakrishna16) [inline lesson8/public/index.html]: you can remove console.log

File: lesson8/public/index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    </div>
+    <script>
+const button = document.getElementById('submitBox');
+button.onclick=()=>{
+  const name = document.getElementById('nameBox').value;
+  const comment = document.getElementById('commentBox').value;
+  const xhttp = new XMLHttpRequest();
+  xhttp.open('POST', '/submit');
+  xhttp.setRequestHeader('content-type', 'application/json');
+  xhttp.send(JSON.stringify({
```
Comment (allopez7) [inline lesson8/public/index.html]: {name, comment} and indent parentheses.

File: lesson8/public/index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  const comment = document.getElementById('commentBox');
+  const canvas = document.getElementById('canvas');
+  const context = canvas.getContext('2d');
+  context.drawImage(video,0,0);
+  const dataURl = canvas.toDataURL('/image.png');
+  const xhttp = new XMLHttpRequest();
+  xhttp.open('POST', '/pics');
+  xhttp.setRequestHeader('content-type', 'application/JSON');
+  xhttp.send(JSON.stringifyfy({
+    canvas: dataURl, name, comment}))
```
Comment (allopez7) [inline lesson8/public/index.html]: do the same here.

**Review by bramakrishna16 — APPROVED** [review]

**Review by allopez7 — APPROVED** [review]
GOOD JOB!!!

**Pull request merged**